tests/acceptance/test_files/plugin.bats

```diff
@@ -73,14 +73,17 @@ load "$LIB_BATS_SUPPORT/load.bash"
 
   # check table output
   run steampipe plugin list --install-dir $MY_TEST_COPY
-  echo $output
```

Comment:
Do not remove the echo $output. This is needed while debugging failing tests.

Reply:
sure, was thinking that it was present for debugging. but didn't see it everywhere so had removed that.